master/docs/manual/configuration/steps/diffinfo.rst

```diff
@@ -0,0 +1,25 @@
+.. bb:step:: DiffInfo
+
+.. _Step-DiffInfo:
+
+DiffInfo
+++++++++
+
+The `DiffInfo` step gathers information about differences between the current revision and the last common ancestor of this revision and another commit or branch.
+This information is useful for various reporters to be able to identify new warnings that appear in newly modified code.
+The diff information is stored as a custom json as transient build data via ``setBuildData`` function.
```

Comment:
we probably need a spec for this data, so that other VCS can generate the same data.

Reply:
Fixed now.